.circleci/config.yml

```diff
@@ -13,6 +13,12 @@ executors:
     resource_class: medium
     environment:
       NODE_OPTIONS: --max_old_space_size=3072
+  node-linux-medium:
+    machine:
+      image: ubuntu-2204:2024.04.4
+    resource_class: medium #// linux medium: 2 CPUs, 7.5 GB RAM, 10 credits/min
+    environment:
+      NODE_OPTIONS: --max_old_space_size=6144
```

Comment:
This is slightly less expensive than docker medium. We don't need the pre-installed browsers, so I think it makes sense.

If we want to include webpack's `zip` for all browser targets we'll either need to optimize the webpack zip integration, or increase the available memory, as `zip` consumes a lot of memory.